### PR TITLE
Enhance Page Not Found

### PR DESCRIPTION
**Pull request by conbrad**
Based on the figma design here (option 3): https://www.figma.com/file/2VuF3FNyhuf5gKD7jqEbeR/MoreCast?node-id=135%3A2

Screen Shot 2021-03-19 at 10 08 38 AM

### CODE REVIEW

**Review by Kyubinhan — COMMENTED** [review]
Can you enable ESlint and Prettier in your IDE and resolve linting issues?

**Comment by conbrad** [comment]
> Can you enable ESlint and Prettier in your IDE and resolve linting issues?

Strange, it's on...

**Review by Kyubinhan — APPROVED** [review]
looks great!

File: web/src/features/fireWeather/pages/NoMatchPage.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+export const NoMatchPage: React.FunctionComponent = () => {
+  return (
+    <main>
+      <PageHeader
+        title="Predictive Services Unit"
+        productName="Predictive Services Unit"
+      />
+      <Container>
+        <h1>Page Not Found</h1>
+        <p>If you enetered a web address please check it was correct.</p>
```
Comment (andrea-williams) [inline web/src/features/fireWeather/pages/NoMatchPage.tsx:13]:
```suggestion
        <p>If you entered a web address please check it is correct.</p>
```

**Review by andrea-williams — APPROVED** [review]
Small typo & change of tense but otherwise all good.

**Pull request merged**